themes/src/theme.js

```diff
@@ -66,8 +66,37 @@ export const setTheme = (_theme, _colors) => {
     }30`,
   };
 
+  let simbaPropsSheetInner = ':root {';
   for (let [key, value] of Object.entries(obj)) {
-    document.documentElement.style.setProperty(key, value);
+    simbaPropsSheetInner += `${key}:${value};\n`;
+  }
+  simbaPropsSheetInner += '}';
+
+  // Constructible stylesheets not supported on Safari
```

Comment:
there is no polyfill for that ?

Reply:
https://www.npmjs.com/package/construct-style-sheets-polyfill there is, but it's too large for something I can workaround with <10 LOC and no degradation perf-wise.

Reply:
I was thinking about a polyfill made by Webreflection but ok.